### DISCUSSION TITLE
How to connect to an Interbase (firebird) DB using Sequel?

### DISCUSSION THREAD

**Discussion by wuarmin** [jeremyevans/sequel#2124] [General]
Hey :wave:,

it seems that the interbase/firebird adapter was removed some time ago? What was the reason, and how to connect to an Interbase DB today?

Thanks!
best regards
Armin

**Comment by jeremyevans** [comment]
The firebird adapter relied on an outdated, unmaintained fork of the driver.  It never passed integration tests, and trying to switch to the official driver made things much worse. For some background, see https://github.com/jeremyevans/sequel/pull/760

Firebird database usage is very small compared to other databases, and I don't think it's worth trying to maintain it.  The firebird adapter was removed along with the db2, dbi, fdbsql, jdbc/fdbsql, informix, and openbase adapters in Sequel 5.

Not sure how to connect to Interbase using Sequel today.  You would probably be better off writing a new adapter from scratch, possibly using the old adapter for reference.

**Reply by wuarmin** [reply]
@jeremyevans Thank you for your explanations. It makes no further sense for me to pursue this at the moment. It's about an Interbase Embarcadero DB. According to my tests, the extension https://github.com/rowland/fb does not seem to work either. I think with Ruby the access is associated with effort, apart from ODBC, which I don't want to use. I just need an ETL. I may have to use python, cause there I found an extension.

thanks
